src/core/Akka.FSharp/Actors.fs

```diff
@@ -0,0 +1,389 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="Actors.fs" company="Akka.NET Project">
+//     Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Akka.FSharp
```

Comment:
Reduce the indenting by changing this to `module Akka.FSharp.Actors` and adding the `[<AutoOpen>]` attribute here. If that causes you to need to move the `OptionHelper` out of the module so that it can be accessed, move it to a `Prelude.fs`.